azure-pipelines.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   - job: CppLint
     pool:
       vmImage: "ubuntu-latest"
-    container: sharaku/cpplint:latest
+    container: helics/buildenv:cpplint
```

Comment:
the container we had been using hasn't been updated in 5 years.  Figured we needed something a little newer.